### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by trotzig**

### CODE REVIEW

**Comment by trotzig** [comment]
I've successfully run import-js in Sublime on Windows 7 (through VirtualBox). It requires changes to the `importjs` binary as well, which I'm currently working on.

File: import-js.py [hunk 1]
```diff
@@ -9,21 +9,24 @@
...
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE
-    ).communicate()[0].decode()
-    path = out.split('__SUBL_PATH__', 2)
-
-    if len(path) > 1:
-        return path[1]
-
-    return ''
+    if 'SHELL' in os.environ:
```
Comment (lencioni) [inline import-js.py:12]: Does sublime linter have this issue? How do they deal with it? If they don't, perhaps you could submit a PR to them as well?

Comment (trotzig) [inline import-js.py:12]: I got this fix from them... Should have mentioned that in the commit message.

Comment (lencioni) [inline import-js.py:12]: 👍

**Pull request merged**